Information/Design.md

```diff
@@ -89,3 +89,320 @@
+held off until the second phase of development. If the backend is change. Thus, currently 
+speaking this is a requirement to design hiccup that comes as an inevitable consequence 
+form the waterfall process.  
+
+
+
+examples of _user_ command usage:
+
+`-user Josh`  -- example with _name_ parameter 
+
+`-user 2`     -- example with _id_ parameter 
+
+
+
+
+------------------------------------------------------------------------------------------
+
+
+######REQ 160 Design:
+
+
+Command name: _-my victims_ 
+
+Parameters: _name_ 
+
+**Design of _user_ command**:
+
+The command _my victims_ takes in the name of a monster in the monster hunter world video
+game, as its only parameter. It installs it into a list of monsters, and then displays 
+the list to the user via discord. If the name of the monster is not a monster in the 
+Monster Hunter World game. Then the Bot prints out an appropriate message to the user. IF
+the command is entered without any name, or parameter. Then command does nothing and prints
+out nothing to the user via Discord. 
+
+When this command is entered, it will first prompt the user for the user's account name. This 
+way each user has its own saved list of monsters killed. If the user does not have a valid 
+account name or id then the monster is not saved in any list, and an appropriate message is 
+displayed to the user via discord. The user will have to authenticate themselves via their 
+password as well. This will be the same password as in the Discord server. This way random 
+people belonging to the server this Monster Hunter World bot is associated with. Cannot 
+edit the list of monsters they have killed without the knowledge of the users pass code.
+
+There are 48 total monsters in the Monster Hunter World video game. Thus the maximum size 
+of any monster list will be 48. The smallest size of course would be if there were no 
+monsters killed, thus size = 0. Being how the upper bound of a monster list is 48. 
+printing out to discord should not cause any significant undefined bandwidth issues. 
+
+Potentially I could have returned a dictionary here instead of a list, however,
+you would have to utilize a value, or key field per cell in a dictionary in the python 
+programming language. Since it is not needed and would only cause of more unneeded data 
+to the user, and in code. It simply is complexity that is not needed. Thus a list or array 
+in the python language is best for this design.  
+
+
+examples of _my victims_ command usage:
+
+`-my victims Kulu-Ya-Ku`  -- example with _name_ parameter 
+
+
+example of the _user entry_ aspect of the _my victims_ command:
+
+`Enter user account name: Josh`
+
+`Enter the password for Josh: ********`
+
+------------------------------------------------------------------------------------------
+
+
+######REQ 170 Design:
+
+
+Command name: _-gone to soon_ 
+
+Parameters: _name_ 
+
+**Design of _user_ command**:
```

Comment:
This is not the 'user' command

Reply:
LOL, I believe that I copy and pasted thinking I would edit the command name to reflect each command sections when authoring this Design.md document. However, I overlooked it, apparently.